frontend/grunt-tasks/grunt-connect.js

```diff
@@ -1,40 +1,40 @@
 module.exports = function (grunt) {
-    
+    var pushState = require('grunt-connect-pushstate/lib/utils').pushState;
```

Comment:
Please use https://www.npmjs.com/package/connect-pushstate instead of grunt-connect-pushstate since the last one has been already deprecated :)